samples/matter/thermostat/src/temperature_measurement/simulated_temp.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "sensor.h"
+#include "temp_sensor_manager.h"
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+constexpr uint16_t kSimulatedReadingFrequency = (60000 / kSensorTimerPeriodMs); /*Change simulated number every minute*/
+
+#if CONFIG_THERMOSTAT_TEMPERATURE_STEP == 0
+constexpr int16_t sMockTemp[] = { 2000, 2731, 1600, 2100, 1937, 3011, 1500, 1899 };
+
+#else
+constexpr int16_t kMockedTemperatureStep = CONFIG_THERMOSTAT_TEMPERATURE_STEP;
```

Comment:
Maybe we don't need `kMockedTemperatureStep` at all and the assigned config can be used directly?